### ISSUE TITLE
Swagger UI and TypeSpec tabs are not accessible using keyboard : A11y_Typespec_Keyboard.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/typespec#10791]
**Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### GitHub Tags:
#A11yTCS; A11yMAS; #A11ySev2; #DesktopWeb; #Win11; #Chrome; # E2E_TypeSpec_Web_May2026; #WCAG2.1.1; # TypeSpec; #Keyboard;
### Issue Subcategory:
Element not reachable via keyboard . 
### User Experience:
Keyboard-only users (including screen reader users and those with motor disabilities) cannot access or interact with the control at all. This creates a complete barrier to that functionality.
### Repro Steps:
Launch url https://typespec.io/openapi/> Swagger UI button.
### Actual Result:
The Swagger UI and TypeSpec tabs on the Docs page are not accessible using keyboard navigation. Users are unable to focus or switch between these tabs using keyboard keys such as Tab or arrow keys.
### Expected Result:
Tabs should be focusable using the keyboard and allow switching using standard keys (Tab/Enter/Arrow keys).
### How to Fix:
Use natively focusable elements (<button>, <a href>, <input>). For custom controls, add tabindex="0" and implement keyboard event handlers for activation. 
### Environment details:
Application Name: TypeSpec 
URL: [typespec.io](https://typespec.io/)
Edition Windows 11 Enterprise
Version 25H2 OS build 26200.8390
Chrome Version 148.0.7778.168 (Official Build) (64-bit)

### Attachment:

<img width="958" height="536" alt="Image" src="https://github.com/user-attachments/assets/b11776f7-5fe5-414e-ac7b-2a4f0360ae80" />

https://github.com/user-attachments/assets/2582a539-8f86-467d-a556-c26e531fdea2

**Comment by kupatkar99** [comment]
#Closed; #A11yFixed;
Verified the issue in below environment:
https://typespec.io/
Chrome: Version 150.0.7871.182 (Official Build) (64-bit)
issue is fixed. hence closing the bug. Please find added attachment for reference.

https://github.com/user-attachments/assets/3be9ebcb-5ba5-451a-ad74-3c48401c901d
